api_tests/nodes/views/test_node_files_list.py

```diff
@@ -432,14 +441,17 @@ def add_github(self):
         self.user.external_accounts.add(oauth_settings)
         self.user.save()
         addon.user_settings = self.user.get_addon('github')
+        addon.external_account = oauth_settings
         addon.save()
         self.project.save()
+        addon.user_settings.oauth_grants[self.project._id] = {oauth_settings._id: []}
+        addon.user_settings.save()
 
     def check_file_order(self, resp):
         previous_file_name = 0
         for file in resp.json['data']:
             int_file_name = int(file['attributes']['name'])
-            assert(int_file_name > previous_file_name, 'Files were not in order')
+            assert int_file_name > previous_file_name, 'Files were not in order'
```

Comment:
Opportunistic fix: this was causing a `pytest.warning`. It was previously just asserting the truthiness of one of two tuples -- either `(True, 'Files were not in order')` or `(False, 'Files were not in order')`, both of which are `True`